changelog.md

```diff
@@ -31,6 +31,7 @@ pre {
 ```
 
 # changelog

```

Comment:
```suggestion
## Changelog

```

Otherwise the text you added below will trigger dropcaps on the first letter, and this will be cleaner than adding custom styling to the page.

Reply:
added